<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Labirinto - Algoritmo A*</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="2">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string>destino</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>FIXAR (clique duplo):</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string>origem</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QTableWidget" name="tableWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="tabKeyNavigation">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <property name="rowCount">
       <number>10</number>
      </property>
      <property name="columnCount">
       <number>25</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menulabirinto">
    <property name="title">
     <string>Labirinto</string>
    </property>
    <addaction name="actionLer"/>
    <addaction name="actionSalvar"/>
    <addaction name="actionGerar"/>
    <addaction name="separator"/>
    <addaction name="actionSair"/>
   </widget>
   <widget class="QMenu" name="menuCaminho">
    <property name="title">
     <string>Caminho</string>
    </property>
    <addaction name="actionCalcular"/>
    <addaction name="actionLimpar"/>
   </widget>
   <addaction name="menulabirinto"/>
   <addaction name="menuCaminho"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLer">
   <property name="text">
    <string>Ler ...</string>
   </property>
  </action>
  <action name="actionSalvar">
   <property name="text">
    <string>Salvar ...</string>
   </property>
  </action>
  <action name="actionGerar">
   <property name="text">
    <string>Gerar</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="actionCalcular">
   <property name="text">
    <string>Calcular</string>
   </property>
  </action>
  <action name="actionLimpar">
   <property name="text">
    <string>Limpar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
